spec/features/idv/doc_auth/ssn_step_spec.rb

```diff
@@ -28,6 +28,12 @@
     expect(page).to have_current_path(idv_doc_auth_ssn_step)
   end
 
-  # it 'prevents a duplicate ssn' do
-  # end
+  it 'does not proceed to the next page if resolution fails' do
+    allow_any_instance_of(Idv::Agent).to receive(:proof).
+      and_return(success: false, errors: {})
+    fill_out_ssn_form_ok
```

Comment:
Why not fill out the form with the known bad SSN here? `123-45-6666` should cause a failure. That'd get you a little more coverage from this feature spec.